CMakeLists.txt

```diff
@@ -1,12 +1,8 @@
-cmake_minimum_required(VERSION 3.12.0)
+cmake_minimum_required(VERSION 3.12...3.24)
 
-cmake_policy(SET CMP0054 NEW)
 set(SKBUILD_LINK_LIBRARIES_KEYWORD PRIVATE)
-
+set(Python_FIND_IMPLEMENTATIONS CPython PyPy)
 set(THREADS_PREFER_PTHREAD_FLAG ON)
-if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-    set(CMAKE_OSX_DEPLOYMENT_TARGET "10.9" CACHE STRING "Minimum OS X deployment version")
-endif()
```

Comment:
This shouldn't be needed, PyPy sets 10.10 now and CPython has been 10.9 for a while. This could be harmful, though, as AS may need this set to 11 & PyPy might need 10.10. So I think it's better to avoid setting it manually. (It was needed when PyPy was 10.7, as setting <10.9 isn't valid these days).